.github/workflows/continuous-integration.yml

```diff
@@ -133,7 +133,9 @@ jobs:
       run: gulp build
 
     - name: Run tox
-      run: tox --parallel auto -e lint,functests
+      # Note we run the func tests backwards here to prove there are no order
+      # dependent tests
+      run: tox -e functests -- tests/functional --reverse
```

Comment:
I don't think there's a way to pass `posargs` only to the functests arg rather than lint as well